scrapers/PaperStreetMedia.yml

```diff
@@ -71,5 +73,5 @@ xPathScrapers:
                 teensloveanal: "Teens Love Anal"
                 teensloveblackcocks: "Teens Love Black Cocks"
                 thickumz: Thickumz
-                    
+
 # Last Updated February 25, 2020
```

Comment:
Need to update the date

Reply:
Done in https://github.com/stashapp/CommunityScrapers/pull/631/commits/af0b1b5557a19073bf2ca7ab3067add125eeb076